src/System.AppContext/src/System/AppContext.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Threading;
+using Windows.ApplicationModel;
+
+namespace System
+{
+    public static class AppContext
```

Comment:
AppContext is in and exposed from mscorlib in coreclr:
https://github.com/dotnet/coreclr/blob/master/src/mscorlib/model.xml#L162

Is that changing?  @AlexGhiondea?